src/selectedtests/app/controllers/project_task_mappings_controller.py

```diff
@@ -55,21 +52,17 @@ def get(
     changed_files: str,
     project: str,
     threshold: Decimal = Decimal(0),
-    evg_api: EvergreenApi = Depends(get_evg),
-    db: MongoWrapper = Depends(get_db),
 ) -> TaskMappingsResponse:
     """
     Get a list of correlated task mappings for an input list of changed source files.
 
-    :param evg_api: Evergreen API client.
-    :param db: The database.
     :param project: The evergreen project.
     :param changed_files: List of source files to calculate correlated tasks for.
     :param threshold: Minimum threshold desired for flip_count / source_file_seen_count ratio
     """
-    evg_project = try_retrieve_evergreen_project(project, evg_api)
+    evg_project = try_retrieve_evergreen_project(project)
     task_mappings = get_correlated_task_mappings(
-        db.task_mappings(), parse_changed_files(changed_files), evg_project.identifier, threshold
+        parse_changed_files(changed_files), evg_project.identifier, threshold
```

Comment:
API just relies on the modules that do this stuff to do it without help, rather than needing to give them their own dependencies.